src/F2_autoreport.py

```diff
@@ -9,6 +9,8 @@
 import json
```

Comment:
meta-comment1 : you could have committed that in 3 separate commits as your commit message consists of three separate conceptual things
meta-comment2 : use infinitive (not past tense) and capitalize the first letter of the commit message (as you would for a normal sentence)

:)

Reply:
Understood. 
Will keep in mind when making commits henceforth

Thanks PF!